kidviz/views.py

```diff
@@ -470,7 +470,7 @@ class ObservationAdminView(LoginRequiredMixin, TemplateView):
 
     def selected_chart(self):
         get = self.request.GET or {}
-        chart_keys = ['chart_v1', 'chart_v2', 'chart_v3',
+        chart_keys = ['chart_v1', 'chart_v2',
```

Comment:
Could you rename all names, ids and tempalete names to keep a consistency in the entire project.